lib/modules/apostrophe-images/lib/cursor.js

```diff
@@ -40,6 +40,65 @@ module.exports = {
       }
     });
 
+    self.addFilter('orientation', {
+      finalize: function() {
+        const orientation = self.get('orientation');
+        if (!orientation) {
+          return;
+        }
+
+        self.and(getCriteria(orientation));
+
+        function getCriteria(orient) {
+          if (orient === 'landscape') {
+            return {'attachment.landscape': true};
+          } else if (orient === 'portrait') {
+            return {'attachment.portrait': true};
+          }
+
+          // Last case is square
+          return {
+            'attachment.square': true
+          };
+        }
+      },
+      safeFor: 'public',
+      launder: function(s) {
+        return self.apos.launder.string(s);
+      },
+      choices: function(callback) {
+        const {choices} = options.module.filters
```

Comment:
How do we feel about these filters not being responsive to other filters? As written, the list of orientations would not respond to the fact that you've already filtered by tag and no images with this tag are landscape. cc @abea @stuartromanek